### PR TITLE
Create a new metadata collection based on Wikidata

### PR DESCRIPTION
**Pull request by ninpnin**
- [x] Rename Sparql query files to *.rq
- [x] Wrap script inside a ```if __name__ == "__main__"``` thingy
- [x] Properly merge changes from main branch
- [x] Fix problems with party metadata
- [ ] Adapt tests to the new folder structure
- [x] Re-run mapping script after compiling new metadata

### CODE REVIEW

**Comment by ninpnin** [comment]
Sidenote: the Sparql query files could be renamed to *.rq for syntax highlighting and so.

File: scripts/wikidata.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+	for _, key, value in df[["wiki_id", "twitter"]].itertuples():
+		if key not in d:
+			d[key] = [value]
+		else:
+			d[key].append(value)
+
+	with open('corpus/twitter.json', 'w') as f:
+		json.dump(d, f, ensure_ascii=False, indent=4)
+
+individual()
```
Comment (ninpnin) [inline scripts/wikidata.py]: wrap inside an if __name__ == "__main__"

**Comment by ninpnin** [comment]
Additionally, the main branch had some new changes that need to be merged to dev before merging dev into main.

**Comment by ninpnin** [comment]
```shell
FileNotFoundError: [Errno 2] No such file or directory: 'corpus/talman.csv'
```

Tests need to be adapted to the new folder structure.

**Comment by rbbby** [comment]
> ```shell
> FileNotFoundError: [Errno 2] No such file or directory: 'corpus/talman.csv'
> ```
> 
> Tests need to be adapted to the new folder structure.

Accidentally removed that file but its restored. But yes tests probably need some fixing still.

**Pull request merged**